Procfile

```diff
@@ -1 +1 @@
-web: cd ekip && python manage.py migrate --settings=config.settings.production --noinput && python manage.py collectstatic --settings=config.settings.production --noinput && waitress-serve --port=$VCAP_APP_PORT config.wsgi:application
+web: cd ekip && python manage.py migrate --settings=config.settings.production --noinput && python manage.py collectstatic --settings=config.settings.production --noinput && waitress-serve --port=$PORT config.wsgi:application
```

Comment:
Is this change intentional? Just want to make sure this isn't creeping in as part of statistics page improvements.